stable/yugaware/values.yaml

```diff
@@ -44,6 +43,7 @@ yugaware:
     enabled: true
     ip: ""
     type: "LoadBalancer"
+    postgres: false # Expose internal postgres as an internal Service
```

Comment:
let's move this to a separate section like 

postgres;
  service:
    enabled: true

similar to the existing yugaware config